### DISCUSSION TITLE
Import markdown files with front-matter

### DISCUSSION THREAD

**Discussion by athanhat** [PostOwl/postowl#38] [Ideas]
Hi Michael and Tom, I had a quick look at your PostOwl web site, it's a great start for user autonomy and independent sharing and communication. I am a great fan of Obsidian and Logseq personal information management applications. The content, notes, is based on markup language with minor differences in format and assets (media files) that are linked/embedded into the notes.

Well I think it will be a great idea to find some synergies between PostOwl and Obsidian/Logseq. The first thing that comes to my mind, I am seriously thinking of becoming a user of PostOwl for this, is a tool to import markdown notes/posts with assets folder(s).

What do you think ?

My best wishes for a successful continuation of your project...

**Comment by keybits** [comment]
Hi @athanhat 😄 

Thanks for getting in touch. I'm an Obsidian user too and absolutely love it. The ability to import a note form Obsidian sounds like a great idea.

Since the initial launch of PostOwl we've taken a bit of a break to focus on some other things. But I'm about to get back into moving PostOwl forwards. The first priority is to make a hosted version for non-technical people. After that, I'll be happy to look into importing things from Obsidian. Or if you'd like to make a start on something, we'd be happy to link any tool you create from the website ec.

Thanks for your interest in PostOwl, let us know how it goes if you try it out.
